test/blas.jl

```diff
@@ -1380,6 +1379,298 @@ end
     @test C ≈ h_C
 end
 
-end
+@testset "xt" begin
+    @testset "trmm! with element type $elty" for elty in [Float32, Float64, ComplexF32] # test broken for ComplexF64
```

Comment:
What part is broken here? Is it a tolerance issue?

Reply:
Some of the off diagonal elements are wrong (and not by a little), I think.

Reply:
I tried this locally and it works now??? Weird.